app/models/super_good/solidus_taxjar/transaction_sync_batch.rb

```diff
@@ -1,3 +1,9 @@
 class SuperGood::SolidusTaxjar::TransactionSyncBatch < ApplicationRecord
   has_many :transaction_sync_logs
+
+  def status
+    return 'processing' if transaction_sync_logs.processing.any?
+    return 'error' if transaction_sync_logs.error.any?
+    return 'success' if transaction_sync_logs.count == transaction_sync_logs.success.count
```

Comment:
Should we include a default state should all of the conditions fail? It looks like we're relying on this method returning a string, because if it doesn't the call below (`batch.status.capitalize`) will throw an error.